app/models/block.rb

```diff
@@ -12,19 +12,17 @@
 
 class Block < ApplicationRecord
   include Paginable
+  include CacheRemovable
 
   belongs_to :account, required: true
   belongs_to :target_account, class_name: 'Account', required: true
 
   validates :account_id, uniqueness: { scope: :target_account_id }
 
-  after_create  :remove_blocking_cache
-  after_destroy :remove_blocking_cache
-
-  private
-
-  def remove_blocking_cache
-    Rails.cache.delete("exclude_account_ids_for:#{account_id}")
-    Rails.cache.delete("exclude_account_ids_for:#{target_account_id}")
+  REMOVE_BLOCKING_CACHE = -> do
```

Comment:
I don't understand why a constant proc is better than a method definition? I agree with abstracting to concern module, but I think the methods should remain methods.

Reply:
Using these constants instead of defining similar methods are not sophisticated enough, but better at least in terms of removing the redundant tests.

Reply:
agreed :+1: